src/lang/sv.js

```diff
@@ -0,0 +1,17 @@
+export const sv = {
+  "send-sec-msg": "Skicka",
+  "sec-success-msg": "Lyckat",
+  "sec-error-msg": "Var god ange till klagomål",
```

Comment:
"Var god ange till klagomål", should probably be something like "Var god ange problemet/klagomålet" or something